simulation_parameters/common/screen_effects.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "none": {
+    "name": "SCREEN_EFFECT_NONE",
+    "index": 0,
+    "ShaderPath": null
+  },
+  "greyscale": {
+    "name": "SCREEN_EFFECT_GREYSCALE",
+    "index": 1,
+    "ShaderPath": "res://shaders/screen_effects/Greyscale.shader"
```

Comment:
This path is wrong and fails on case sensitive filesystems (which I think exported Godot games also use). I'll fix this as well.